extensions/amp-ooyala-player/0.1/amp-ooyala-player.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {user} from '../../../src/log';
+import {removeElement} from '../../../src/dom';
+
+class AmpOoyalaPlayer extends AMP.BaseElement {
```

Comment:
We now expect all new video-players to implement our video-interface. 

- Please see https://github.com/ampproject/amphtml/blob/master/src/video-interface.js for the methods, attributes and events players are expected to implement.
- Please see https://github.com/ampproject/amphtml/blob/master/builtins/amp-video.js for an example of a video player that fully implements this interface.
- Please see https://github.com/aghassemi/amphtml/blob/video-yt/extensions/amp-youtube/0.1/amp-youtube.js for an iframe-based player that implements this interface (Note this code is WIP and might be out of date)